nexus/src/db/error.rs

```diff
@@ -210,8 +220,8 @@ fn public_error_from_diesel_create(
             )),
         },
         _ => PublicError::internal_error(&format!(
-            "Unknown diesel error: {:?}",
-            error
+            "Unknown diesel error creating {:?} called {:?}: {:#}",
+            resource_type, object_name, error
```

Comment:
Did these lines end up being useful? I think the change is probably good regardless, so I'm just curious if you ran into another opaque error message that this helped resolve.

Reply:
No, I didn't wind up using these.